src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -349,12 +351,16 @@ private bool OnScroll(object sender, RoutedEventArgs routedEventArgs)
 			// as for finger and pen we will get a PointerCancelled which will reset the pressed state to false.
 			// And it would also requires us to explicitly invoke OnScroll in PointerRelease in order to raise the
 			// final SV.ViewChanged event with a IsIntermediate == false.
```

Comment:
```suggestion
			// as for finger and pen we will get a PointerCancelled which will reset the pressed state to false.
			// And it would also require us to explicitly invoke OnScroll in PointerRelease in order to raise the
			// final SV.ViewChanged event with an IsIntermediate == false.
```